src/matlab2tikz.m

```diff
@@ -1366,6 +1367,18 @@ case guitypes()
         end
         str = join(m2t, str, '\\[1ex]');
         opts =  opts_add(opts, tikzKeyword, sprintf('{%s}', str));
+    
+        % set color of axis labels
+        color = object.Color;
+        [m2t, col] = getColor(m2t, handle, color, 'patch');
+        if strcmpi(labelKind,'title')
+          % TODO: color titles
+        else
+          opts = ...
+              opts_add(opts, ...
+              ['every axis ',lower(labelKind(1)),' label/.append style'], ...
+              ['{font=\color{',col,'}}']);
+        end
```

Comment:
@patrickkwang :+1:, but indeed, `color = object.Color` should be replaced by `color = get(object, 'Color')` for backwards compatibility.